modules/gapi/src/backends/fluid/gfluidimgproc_func.hpp

```diff
@@ -52,7 +53,7 @@ void run_rgb2yuv_impl(uchar out[], const uchar in[], int width, const float coef
 
 void run_yuv2rgb_impl(uchar out[], const uchar in[], int width, const float coef[4]);
 
-void run_rgb2yuv422_impl(uchar out[], const uchar in[], int width);
+void GAPI_EXPORTS run_rgb2yuv422_impl(uchar out[], const uchar in[], int width);
```

Comment:
It is used by serialization backend imgproc kernels collection. I can minimize number of kernels there. It will be quite big cleanup effort. Just to leave only kernels used by tests. Then we can remove it as well.